### PR TITLE
Cb 1 explanation

### PR DESCRIPTION
**Pull request by Aarushi-Pandey**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Creates a new component called Explanation which renders a title, subtitle and a short explanation of the site using Jumbotron.

* **What is the current behavior?What does the code do before you made the new changes?** 
Before, it had nothing (didn't exist).

* **What is the new behavior?**
It opens the Explanation page. 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No changes should be needed. 

* **Other information**:

### CODE REVIEW

File: frontend/src/components/LandingPage/Explanation/Explanation.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//Import components that you create here
+
+function Explanation(){
+    return(
+        //Use components that you import here
+        <Jumbotron>
+        <h1 className="display-3">Explanation</h1>
+        <p className="lead">What does Explorezy do?</p>
+        <hr className="my-2"></hr>
+        <p>Want to explore new places but COVID-19 happened? Try this website, which will let you explore places virtually in a fun way! Enter your zip code, choose a radius, and get a list of cities near you within that radius. If there's any city you want to visit, just click on it and get tons of information about fun things to do in that city. Along with popular tourist spots, you'll receive information about transportation to those places, flights (in case you want to visit in real life!) and restaurants nearby. (Basically we'll plan your whole trip out for you for free!) Happy travelling!!</p>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Explanation/Explanation.js]: What do you think about adding some new lines here to make this a little bit more readable?

File: frontend/src/components/LandingPage/Explanation/Explanation.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Jumbotron } from 'reactstrap'
+//Import components that you create here
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Explanation/Explanation.js]: Is this a good comment to have here?

File: frontend/src/components/LandingPage/Explanation/Explanation.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+function Explanation(){
+    return(
+        //Use components that you import here
+        <Jumbotron>
+        <h1 className="display-3">Explanation</h1>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Explanation/Explanation.js]: What do you think about indenting over the elements that are inside of the jumbotron for readability?

**Review by Codubee — DISMISSED** [review]
Great job Aarushi! Left some comments that we can talk about.

File: frontend/src/components/LandingPage/Explanation/Explanation.js (hunk 1 above)
Comment (Aarushi-Pandey) [reply frontend/src/components/LandingPage/Explanation/Explanation.js]: I think that's a great idea! I'll add new lines asap.

File: frontend/src/components/LandingPage/Explanation/Explanation.js (hunk 2 above)
Comment (Aarushi-Pandey) [reply frontend/src/components/LandingPage/Explanation/Explanation.js]: Oops, forgot to remove some comments. Will try to add some of my own instead.

File: frontend/src/components/LandingPage/Explanation/Explanation.js (hunk 3 above)
Comment (Aarushi-Pandey) [reply frontend/src/components/LandingPage/Explanation/Explanation.js]: Good idea, indenting is imp for sure.

**Review by JerryTeng01 — APPROVED** [review]
Hi Aarushi! Your component looks very nice! I think it might look nicer aesthetically if we made the paragraph into a bullet style format so that it's easier to read than a large chunk of text. Instead of having "Explanation" at the top, you could rephrase it as "What is Explorezy?" and remove the subtitle to make it look more like a real app. Another idea is you could add some images on the left and right sides with the bulleted text in the center. Just some food for thought :octocat: . Looks good, keep up the great work!

**Review by sulemanbutta — APPROVED** [review]
Hi Aarushi, your code looks good to me! I do agree with Jerry on adding bullet points and removing subtitles, but in addition I suggest you should maybe remove the parentheses around "(Basically we'll plan your whole trip out for you for free!) " and remove "(in case you want to visit in real life!)" . It just seems a bit redundant to me. Good job regardless.

**Review by Codubee — CHANGES_REQUESTED** [review]
LGTM! Great job Aarushi!

**Review by Codubee — APPROVED** [review]
Meant to approve sorry!

**Comment by Codubee** [comment]
There are some merge conflicts that you have to resolve before I can merge. I can help you resolve these if youd like.

**Comment by Aarushi-Pandey** [comment]
I added some pictures after my explanation and converted my explanation into bullet points. Thanks for the suggestions Jerry and Suleman!

**Pull request merged**